From bc7992c483a5a188b893a0f0a23d9205274a4485 Mon Sep 17 00:00:00 2001
From: Romain Naour <romain.naour@gmail.com>
Date: Sat, 1 Aug 2020 01:38:24 +0200
Subject: [PATCH] doc/meson.build: add single quote for env

Meson trigger an error due to missing sigle quote for env:

efl-1.24.3/doc/meson.build:114:2: ERROR: Unknown variable "env"

Upstream status:
https://sourceforge.net/p/enlightenment/mailman/message/37075003/

Signed-off-by: Romain Naour <romain.naour@gmail.com>
---
 doc/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/meson.build b/doc/meson.build
index 5ad389828c..11e2b6446b 100644
--- a/doc/meson.build
+++ b/doc/meson.build
@@ -113,7 +113,7 @@ foreach text_filter_property : text_filter_properties
 
   doc_target += custom_target('preview_text_filters_'+name,
                   command: [
-                    env, 'EFL_RUN_IN_TREE=1',
+                    'env', 'EFL_RUN_IN_TREE=1',
                     'EFL_EVAS_FILTER_LUA_PREFIX=' + join_paths(meson.source_root(), 'src', 'lib', 'evas'),
                     preview_text_filter.full_path(), text, filter_code.stdout(), '@OUTPUT@', font, size
                   ],
-- 
2.25.4

